CONTRIBUTING.md

```diff
@@ -11,6 +11,12 @@ of Anki.
 If you want to add new library dependency - add them to the `Pipfile`.  
 The dependency management is implemented using https://pipenv.org/
 
+## UI Files
+The config is build using QTDesigner, which you can get by running `sudo apt install qtcreator` on Linux, or simply download from `https://build-system.fman.io/qt-designer-download`
+
+In QT Designer you can create UI files using a simple GUI, then use the `pyuic5` module to convert them to python:
+`pyuic5 ui_files/config.ui -o crowd_anki/config/config_ui.py`
```

Comment:
this should be part of build/packaging process (see shell script in the root)

Reply:
https://github.com/Stvad/CrowdAnki/blob/master/package_plugin.sh

Reply:
Oh yes, that's a good idea, I'll add it there.

In the future if more UI files are added then switching over to using [Glutanimate's AnkiAddonBuilder](https://github.com/glutanimate/anki-addon-builder) should be considered. It builds all the UI files in the repo for you, and packages the add-on for both(/either) Anki 2 and 2.1. I have no idea if there is some dependency which means this project cannot use that, and I am also not wedded to the idea, it's just a thought :thinking:

Reply:
also pyuic5 should be added to build dependencies

Reply:
I believe this one needs to be updated to reflect changes in automation around it :)

Reply:
Oh yes, I updated the documentation too.

Reply:
👍 It bothers me that people may need to copypaste things from here though and more details then I like are exposed 🤔.

How about moving this line invocation into a separate script like `generate_ui.sh` that would install the pipenv dev deps (to get the pyuic5) and runs this. Or maybe just runs this, depending on whether repeatedly running pipenv takes long time.

And mentioning it here + using it from the other scipt?

+ should probably mention installing pipenv dev deps as a way to get the `pyuic5` in general

Also it's `QtDesigner` with small `t` 😛

Reply:
Good call, changed :+1: added `generate_ui.sh`, called it from the packaging script, and rewrote the documentation :+1: